pupil_src/launchables/eye.py

```diff
@@ -213,19 +213,21 @@ def get_timestamp():
         ]
         if eye_id == 0:
             preferred_names += ["HD-6000"]
-        default_capture_settings = (
-            "UVC_Source",
-            {
-                "preferred_names": preferred_names,
-                "frame_size": (320, 240),
-                "frame_rate": 120,
-            },
-        )
+
+        default_capture = "UVC_Source"
```

Comment:
`<nitpick>`
I'd suggest `default_capture_name` to be more explicit, and symmetry. Feel free to ignore this suggestion :)
`</nitpick>`

Reply:
Makes total sense!